manifests/init.pp

```diff
@@ -116,9 +116,12 @@
 
   Service['httpd'] -> Exec['foreman-rake-db:seed']
 
+  class { 'katello::plugin': }
+
   if $katello::gutterball {
```

Comment:
Can we move this chunk and logic in plugin/gutterball.pp ? Just makes more sense to all live together for me.